### PR TITLE
[Broadcom]: Update the docker-syncd-rpc Makefile and Dockerfile

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: platform/broadcom/docker-syncd-brcm-rpc.mk [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 DOCKER_SYNCD_BRCM_RPC = docker-syncd-brcm-rpc.gz
 $(DOCKER_SYNCD_BRCM_RPC)_PATH = $(PLATFORM_PATH)/docker-syncd-brcm-rpc
-$(DOCKER_SYNCD_BRCM_RPC)_DEPENDS += $(SYNCD_RPC) $(LIBTHRIFT)
+$(DOCKER_SYNCD_BRCM_RPC)_DEPENDS += $(SYNCD) $(LIBTHRIFT)
```
Comment (lguohan) [inline platform/broadcom/docker-syncd-brcm-rpc.mk]: is this a temp fix?

File: platform/broadcom/docker-syncd-brcm-rpc/Dockerfile.j2 [hunk 2]
```diff
@@ -9,8 +9,6 @@ debs/{{ deb }}{{' '}}
 {%- endfor -%}
 debs/
 
-RUN dpkg -P syncd
```
Comment (lguohan) [inline platform/broadcom/docker-syncd-brcm-rpc/Dockerfile.j2]: +# FIXME: Swap SYNCD to SYNCD_RPC after libsaithrift is compatible with SAI v1.0

File: platform/broadcom/docker-syncd-brcm-rpc/Dockerfile.j2 [hunk 3]
```diff
@@ -9,13 +9,14 @@ debs/{{ deb }}{{' '}}
...
-
 RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; } ; \
 {% for deb in docker_syncd_brcm_rpc_debs.split(' ') -%}
 dpkg_apt debs/{{ deb }}{{'; '}}
 {%- endfor %}
 
+# FIXME: Uncomment this line after libsaithrift is compatible with SAI v1.0
+# RUN dpkg -P syncd
+
```
Comment (pavel-shirshov) [inline platform/broadcom/docker-syncd-brcm-rpc/Dockerfile.j2]: It's better to move this purge operation on previous place. We should remove old syncd before we install new one

**Comment by stcheng** [comment]
I will fix it in a separate pull request.

**Pull request closed without merging**